src/models/fhir/types.ts

```diff
@@ -0,0 +1,57 @@
+import { R4 } from "@ahryman40k/ts-fhir-types";
```

Comment:
this is an internal representation, not really fhir right? (a bit confusing to be in a fhir model if that's the case)

Reply:
Yes, `src/models/fhir/types.ts` is meant to provide internal representation.

However, I still think it's useful to import R4 from `@ahryman40k/ts-fhir-types` so we can utilise some of their interfaces like `PatientGenderKind` since it is also what we are expecting to receive in a valid FHIR resource.

```typescript
export interface Patient {
  gender?: R4.PatientGenderKind;
  // ...
}
```

Reply:
I'm fine with `gender?: R4.PatientGenderKind;`. I meant that the location of the file make it look like it's not an internal representation but expected FHIR objects

Reply:
Oh I see. Shall I rename the filename to something like:
- `src/models/parsedFhir/types.ts`
- `src/models/parsedResource/types.ts`
- etc.